CONTRIBUTING.md

```diff
@@ -41,60 +47,57 @@ potentially received objections/suggestions to be handled.
 
 Once the design has been finalized in the Google Doc, the proposed set of
 changes to the specification should be made available for review. This could
-be done by pointing to a branch in a Github repo with the proposed edits.
-Reviewers, or implementers of the feature, can then easily see the exact
-changes being proposed and comment on them within that branch.
-
-Before a formal PR for the changes is created, the feature needs to be
-validated through a full implementation by at least one platform that is
-easily accessible to reviewers. Note that in many cases this would include
-support for the proposed changes by one or more service brokers
-(potentially samples). How each platform makes this new feature available
-will vary between platforms. It is strongly recommended that as part of the
-design review process platforms that plan on implementing the feature
-get confirmation from the reviewers that their planned mechanism for providing
-access to the implementation of the feature is acceptable.
-
-It is expected that during this implementation phase there will be changes
-made to the design and proposed specification edits to accurately represent
-the current status of the proposal.
-
-If the proposal adds or changes an object model or resource endpoint definition,
-it is expected that the PR will also include the necessary updates to the
-[Open API document](openapi.yaml).
+be done by pointing to a branch in a Github repo with the proposed edits or by
+creating a PR with the `do not merge` label assigned to it. Reviewers, or
+potential implementers of the feature, can then easily see the exact changes
+being proposed so that they can comment on them. If the proposal adds or changes
+an object model or resource endpoint definition, it is expected that the PR will
+also include the necessary updates to the [Open API document](openapi.yaml) and
+[Swagger document](swagger.yaml).
 
 Once support for the change has been implemented, a PR should be sent to this
```

Comment:
"has been implemented" - by who/how? We don't talk about validation until later so this feels like we're missing something in the flow.

Reply:
Good point. I've combined a couple of sections to split into just "Minor Changes" and "Major Changes". Let me know what you think...